pandas/tests/io/test_html.py

```diff
@@ -3,13 +3,17 @@
 import glob
 import os
 import re
+import threading
 import warnings
 
+
+# imports needed for Python 3.x but will fail under Python 2.x
```

Comment:
@3553x : He means to just remove the line.  I thought that line would be useful for clarity?